CONTRIBUTING.md

```diff
@@ -48,8 +40,132 @@ See [LICENSE](LICENSE).
 * [ ] I am familiar to proper capitalization for project-specific terminology.
 See [Capitalization](#Capitalization).
 * [ ] The first line of the file contains the modular docs content type attribute, for example, `:_mod-docs-content-type: ASSEMBLY` for assemblies.
+* [ ] I fill out the cherry-picking list in the PR template to the best of my abilities to signify which versions my update applies to.
+If unsure, I let reviewers know so that they can assist.
+
+## Foreman documentation minimalist style guide
```

Comment:
Potential reviewers: Note that all these style conventions have been moved from another document. I didn't write them and I'd like to avoid tweaking how they are worded :upside_down_face: If anyone would like to see the wording improved, I'd suggest doing it in another PR. Thanks for understanding!

However, as noted in the PR description, I'd be happy to remove lines that are obsolete or redundant.